src/features/search/pages/Search/Search.native.test.tsx

```diff
@@ -232,38 +234,59 @@ describe('<Search/>', () => {
     expect(mockDispatch).not.toHaveBeenCalled()
   })
 
-  it('should setPlace and setLocationMode in location context when there is a place in the uri params', async () => {
-    useRoute.mockReturnValueOnce({
-      params: {
-        locationFilter: {
-          locationType: LocationMode.AROUND_PLACE,
-          place: mockedPlace,
+  describe('When URI params contains', () => {
+    it('a place, it should setPlace and setLocationMode in location context', async () => {
```

Comment:
Petite quack dans le titre 'it a place...' je ne suis pas sûr de ce que tu as voulu dire